Gemfile

```diff
@@ -60,20 +59,21 @@ gem 'db2fog'
 gem 'haml'
 gem 'rabl'
 gem 'redcarpet'
-gem 'sass', "~> 3.3"
-gem 'sass-rails', '~> 3.2.3'
+gem 'sass'
+gem 'sass-rails'
 gem 'truncate_html'
 gem 'unicorn'
 
 # AMS is pinned to 0.8.4 because 0.9.x is a complete re-write, as is 0.10.x
 # Once Rails is updated to 5.x we should bump directly to 0.10.x
 gem "active_model_serializers", "0.8.4"
+gem 'activerecord-session_store'
 gem 'acts-as-taggable-on', '~> 3.4'
 gem 'angularjs-file-upload-rails', '~> 2.4.1'
 gem 'blockenspiel'
 gem 'custom_error_message', github: 'jeremydurham/custom-err-msg'
 gem 'dalli'
-gem 'deface', '1.0.2'
+gem 'deface'
```

Comment:
I guess we can trust all versions since it's maintained by the Spree team.

Reply:
we can remove deface.